roles/ceph-mon/tasks/openstack_config.yml

```diff
@@ -1,15 +1,13 @@
 ---
 - name: create openstack pool(s)
-  command: "{{ docker_exec_cmd }} ceph --cluster {{ cluster }} osd pool create {{ item.name }} {{ item.pg_num }} {{ item.rule_name|default('') }}"
+  command: "{{ docker_exec_cmd }} ceph --cluster {{ cluster }} osd pool create {{ item.name }} {{ item.pg_num }} {{ item.rule_name|default('') }} {{ item.size|default('') }}"
```

Comment:
The only problem I see here is that `item.rule_name` can be empty and `item.size` can be populated. This will likely result in an error from the Ceph CLI.